applications/launchpad/gui-react/src/utils/Object.ts

```diff
@@ -0,0 +1,47 @@
+const ObjectUtils = {
```

Comment:
of all the code in here, this is what I would like to have tests for, it's too convoluted to leave without tests

Reply:
You are right! I've switched to the `deepmerge` package, so this file has been removed.